OpenRA.Game/Settings.cs

```diff
@@ -90,18 +90,36 @@ public class DebugSettings
 		public bool SanityCheckUnsyncedCode = false;
 		public int Samples = 25;
 		public bool IgnoreVersionMismatch = false;
+
+		[Desc("Dump performance data into cpu.csv and render.csv accessible via ~/.openra/Logs.")]
+		public bool Benchmark = false;
```

Comment:
I don't think it is appropriate to include this as a settings flag because it doesn't have any UI and it has its own performance impact.  Please implement this as a launch argument instead.